### PR TITLE
[Gene] Scroll locking

### PR DESCRIPTION
**Pull request by orta**
Oooook, that's got one of the trickiest bits in, handling the scroll locking inside the scroll

These changes are super documented, so it's better to read the source than for me to re-write it here.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG added.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/2a53424c9340d7da56dd2d586e9a75e1677b104d/lib/containers/gene.js">lib/containers/gene.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: lib/containers/gene.js [hunk 1]
```diff
@@ -25,14 +25,42 @@ const TABS = {
...
+   *   - The sticky refine, when scrolled up _gains_ a 64px margin
+   *     this is so it can reach all the way of the screen, and fit
+   *     the sticky header's mini title inside it.
+   *
+   *   - We use a fork of react-native-parallax-scroll-view which has access
+   *     to change the style component of the header, as well as a well-ordered
+   *     API for inserting a component into the tree. This is used so that the
+   *     sticky refine section will _always_ be at a specific index, making sure
+   *     the `stickyHeaderIndices` is always at the right index.
+   *
```
Comment (mennenia) [inline lib/containers/gene.js]: Awesome comment

File: package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "lodash": "3.10.1",
     "react": "15.3.2",
     "react-native": "0.34.1",
-    "react-native-parallax-scroll-view": "^0.19.0",
+    "react-native-parallax-scroll-view": "https://github.com/orta/react-native-parallax-scroll-view",
```
Comment (mennenia) [inline package.json]: 👍

**Comment by mennenia** [comment]
I liiiike!


Useful component, parallax scroll view :o.

**Pull request merged**